src/services/RealtimeService.ts

```diff
@@ -212,7 +208,7 @@ export class RealtimeService {
     this.ws.onerror = (e: any) => {
       console.error(`[RealtimeService ${this.id}] got error on websocket client, closing: ${e.message}`);
       if (this.ws) this.ws.close();
-      // this.setState("closed"); // TODO: should this be set?
+      this.setState("closed");
```

Comment:
A little confused what the different states mean (Closed vs disconnecting vs failed, does an error mean something failed for instance?), but if closed is correct here then 👍